packages/gp2-components/src/organisms/UserDetailHeaderCard.tsx

```diff
@@ -1,14 +1,21 @@
 import { gp2 as gp2Model } from '@asap-hub/model';
-import { Avatar, Headline3, Link, pixels } from '@asap-hub/react-components';
+import {
+  Avatar,
+  Headline3,
+  Link,
+  pixels,
+  uploadIcon,
+} from '@asap-hub/react-components';
 import { css } from '@emotion/react';
-import { addIcon } from '../icons';
+import { addIcon, editIcon } from '../icons';
 import locationIcon from '../icons/location-icon';
 import roleIcon from '../icons/role-icon';
 import { usersHeaderImage } from '../images';
 import { mobileQuery, nonMobileQuery } from '../layout';
 import CardWithBackground from '../molecules/CardWithBackground';
```

Comment:
Merge these imports into one, if possible
```suggestion
import { CardWithBackground, IconWithLabel, UserRegion} from '../molecules';
```